packages/token-list/src/index.ts

```diff
@@ -4,49 +4,45 @@
  * @see https://dom.spec.whatwg.org/#domtokenlist
  */
 export default class TokenList {
+	private _currentValue: string;
+	private _valueAsArray: string[];
+
 	/**
 	 * Constructs a new instance of TokenList.
 	 *
 	 * @param {string} initialValue Initial value to assign.
```

Comment:
All the types that have moved into TypeScript syntax should be removed from JSDoc annotations like this:

```suggestion
	 * @param initialValue Initial value to assign.
```